src/QAT/qat/purr/compiler/builders.py

```diff
@@ -45,6 +46,12 @@
 log = get_default_logger()
 
 
+class Axis(Enum):
```

Comment:
Not used anywhere I can see?

Reply:
https://github.com/oqc-community/qat/pull/49/files#diff-ce42f535a52f24404f7a982ad24a8b40e2057434b056ffc9c56e32625ddc1e16R54